mlir/utils/performance/hipblaslt-benchmark-driver/verify_hipblaslt.py

```diff
@@ -0,0 +1,234 @@
+#!/usr/bin/env python3
+"""
+Verification helper for hipblaslt-benchmark-driver.
+
+This script compares the output of hipblaslt-benchmark-driver against
+rocmlir-gen by running both with matching GEMM parameters and comparing
+the printed results.
+
+Usage:
+    python verify_hipblaslt.py -m 32 -n 32 -k 32 -t f32 -arch gfx942 \\
+        --hipblaslt-path <path> \\
+        --rocmlir-gen-path <path> \\
+        --rocmlir-driver-path <path> \\
+        --runner-path <path> \\
+        --libs <shared_libs>
+"""
+
+import argparse
+import re
+import subprocess
+import sys
+
+
+def parse_tensor_output(output):
+    """Parse the printed tensor values from output.
+
+    Extracts numeric values from the 'data = ' section of printMemrefF32 output.
+    Returns a list of floats.
+    """
+    # Find the data section
+    match = re.search(r'data\s*=\s*\n?(.*)', output, re.DOTALL)
+    if not match:
+        return []
+
+    data_str = match.group(1)
+    # Extract all numbers (including negative and scientific notation)
+    numbers = re.findall(r'[-+]?(?:\d+\.?\d*|\.\d+)(?:[eE][-+]?\d+)?', data_str)
+    return [float(n) for n in numbers]
+
+
+def run_hipblaslt(hipblaslt_path, m, n, k, g, dtype, transA, transB):
+    """Run hipblaslt-benchmark-driver with --print-results."""
+    cmd = [
+        hipblaslt_path,
+        '-m', str(m),
+        '-n', str(n),
+        '-k', str(k),
+        '-g', str(g),
+        '-t', dtype,
+        f'-transA={transA}',
+        f'-transB={transB}',
+        '--kernel-repeats', '1',
+        '--warmup-runs', '1',
+        '--print-results',
+    ]
+
+    try:
+        result = subprocess.run(cmd, capture_output=True, text=True, timeout=120)
+        if result.returncode != 0:
+            return None, f"hipblaslt failed: {result.stderr}"
+        return result.stdout, None
+    except subprocess.TimeoutExpired:
+        return None, "hipblaslt timed out"
+    except Exception as e:
+        return None, str(e)
+
+
+def run_rocmlir_gen(rocmlir_gen_path, rocmlir_driver_path, runner_path, libs,
+                    m, n, k, g, dtype, transA, transB, arch):
+    """Run rocmlir-gen with -pr and execute to get reference output."""
+    # Map dtype to rocmlir-gen format
+    dtype_map = {'f32': 'f32', 'f16': 'f16', 'bf16': 'bf16', 'i8': 'i8', 'fp8': 'fp8'}
+    rocmlir_dtype = dtype_map.get(dtype, dtype)
+
+    # Convert transA/transB to lowercase for rocmlir-gen
+    trans_a = transA.lower() if isinstance(transA, str) else str(transA).lower()
+    trans_b = transB.lower() if isinstance(transB, str) else str(transB).lower()
```

Comment:
```suggestion
def run_rocmlir_gen(rocmlir_gen_path, rocmlir_driver_path, runner_path, libs,
                    m, n, k, g, dtype, trans_a, trans_b, arch):
    """Run rocmlir-gen with -pr and execute to get reference output."""
    # Map dtype to rocmlir-gen format
    dtype_map = {'f32': 'f32', 'f16': 'f16', 'bf16': 'bf16', 'i8': 'i8', 'fp8': 'fp8'}
    rocmlir_dtype = dtype_map.get(dtype, dtype)

    # Convert trans_a/trans_bto lowercase for rocmlir-gen
    trans_a = trans_a.lower() if isinstance(trans_a, str) else str(trans_a).lower()
    trans_b = trans_b.lower() if isinstance(trans_b, str) else str(trans_b).lower()
```
but check since we already use trans_a, so that we don't make ambiguous variables names